### PR TITLE
Ran clang-format on all of one-shot OD

### PR DESCRIPTION
**Pull request by shantanuchhabra**

### CODE REVIEW

File: src/toolkits/object_detection/one_shot_object_detection/CMakeLists.txt [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-    util/parameter_sampler.cpp
-    util/superposition.cpp
-    one_shot_object_detector.cpp
-  REQUIRES
-    image_io
-    random
-    table_printer
-    unity_core
-)
+    make_library(unity_one_shot_object_detection OBJECT SOURCES
```
Comment (znation) [inline src/toolkits/object_detection/one_shot_object_detection/CMakeLists.txt]: Let's revert the changes in this file - I'm not sure clang-format got it right.

Comment (nickjong) [reply src/toolkits/object_detection/one_shot_object_detection/CMakeLists.txt]: Yes, in fact, this no longer builds...

Comment (shantanuchhabra) [reply src/toolkits/object_detection/one_shot_object_detection/CMakeLists.txt]: Fixed.
